docs/src/pages/components/slider/CustomizedSlider.tsx

```diff
@@ -54,20 +56,18 @@ const marks = [
   },
 ];
 
-const IOSSlider = withStyles({
-  root: {
-    color: '#3880ff',
-    height: 2,
-    padding: '15px 0',
-  },
-  thumb: {
+const IosSlider = styled(Slider)({
```

Comment:
The capitalization here is interesting, is IOS vs Ios 🤷‍♂️

Reply:
let's go with IOS, looks better :))